edgepi-adc.html

```diff
@@ -1,251 +1,279 @@
 <script type="text/javascript">
-	RED.nodes.registerType('adc',{
-		category: 'EdgePi',
-		color: '#f391aa',
-		defaults: {
-		name: {value: ""},
-    transport: { value: "Local" },
-    tcpAddress: { value: "" },
-    tcpPort: { value: "" },
-		read: { value: "voltage" },
-    adc: { value: "ADC_1"},
-    adc1DataRate: { value: "SPS_38400"},
-    adc2DataRate: { value: "SPS_800"},
-		channel: { value: "AIN1"},
-    diff: { value: "DIFF_1"},
-  	},
-		inputs:1,
-		outputs:1,
-	  icon: "right-arrow-bracket.svg",
-		label: function() {
-			return this.name||"adc";
+  RED.nodes.registerType("adc", {
+    category: "EdgePi",
+    color: "#f391aa",
+    defaults: {
+      name: { value: "" },
+      transport: { value: "Local" },
+      tcpAddress: { value: "" },
+      tcpPort: { value: "" },
+      read: { value: "voltage" },
+      adcNum: { value: 1 },
+      adc1DataRate: { value: 38400 },
+      adc2DataRate: { value: 800 },
+      channel: { value: 1 },
+      diff: { value: 1 },
     },
-    oneditprepare: function(){
-      // Query html elements
-      const configTransportSelect = document.getElementById("node-input-transport");
+    inputs: 1,
+    outputs: 1,
+    icon: "right-arrow-bracket.svg",
+    label: function () {
+      return this.name || "adc";
+    },
+    oneditprepare: function () {
+      const configTransportSelect = document.getElementById(
+        "node-input-transport"
+      );
       const tcpField = document.querySelector(".form-row.tcp");
-			const read = document.getElementById("node-input-read");
-      const adcOptions = document.getElementById("node-input-adc");
-			const adc1 = document.getElementById("adc-option-adc1");
-      const dataRateField = document.querySelector(".form-row.data-rate")
+      const read = document.getElementById("node-input-read");
+      const adcNum = document.getElementById("node-input-adcNum");
+      const adc1 = document.getElementById("adc-option-adc1");
+      const dataRateField = document.querySelector(".form-row.data-rate");
       const adc1DataRate = document.getElementById("node-input-adc1DataRate");
       const adc2DataRate = document.getElementById("node-input-adc2DataRate");
-			const channelField = document.querySelector(".form-row.channel");
+      const channelField = document.querySelector(".form-row.channel");
       const diffField = document.querySelector(".form-row.diff");
```

Comment:
make sure these fields match what we have in the readme.

Reply:
Updated